app/forms/employment_form.rb

```diff
@@ -37,6 +37,7 @@ class EmploymentForm < Form
   validates :new_job_start_date, date: true
   validates :notice_period_end_date, date: true
   validate :end_date_before_start_date?
+  validate :notice_period_end_date_before_start_date?
```

Comment:
The name of this reads the wrong way around - it reads as "Validate that the notice period end date is before the start date"